content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -7,46 +7,106 @@ content_template: templates/task
 
 {{% capture overview %}}
 
-This page explains how to manage certificates manually with kubeadm.
+Client certificates generated by [kubeadm](/docs/reference/setup-tools/kubeadm/kubeadm/) expire after 1 year. This page explains how to manage certificate renewals with kubeadm.
 
 {{% /capture %}}
 
 {{% capture prerequisites %}}
 
-These are advanced topics for users who need to integrate their organization's certificate infrastructure into a kubeadm-built cluster. If kubeadm with the default configuration satisfies your needs, you should let kubeadm manage certificates instead.
+Be familiar with [PKI certificates and requirements in Kubernetes](/docs/setup/certificates/).
 
-You should be familiar with [PKI certificates and requirements in Kubernetes](/docs/setup/certificates/).
+Have a working Kubernetes cluster installed using kubeadm with your certificates stored in `/etc/kubernetes/pki` folder; in case you are using a different location, the following can be adapted accordingly.
 
```

Comment:
I would suggest a pre-req that requires "kubeadm 1.15 or later," alternately you can add in, at the top, something like:
```
{{< feature-state for_k8s_version="v1.15" state="stable" >}}
```

Reply:
Done, at the beginning of the doc